src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -42,17 +42,20 @@ class TextWrapper
 {
     private $app;
     private $env;
+    private $config_path;
+    private $parser;
     private $exit_code;
 
     /**
-     * @param  Phinx\Console\PhinxApplication $app
-     * @param  String                         $default_env  environment fallback, defaults to "development"
-     * @return void
+     * @param PhinxApplication $app
+     * @param array $options
      */
-    public function __construct(PhinxApplication $app, $default_env = 'development')
+    public function __construct(PhinxApplication $app, Array $options = array())
     {
         $this->app = $app;
-        $this->env = $default_env;
+        $this->env = (isset($options['environment']) ? $options['environment'] : 'development');
```

Comment:
This was done so that we could predict what is being passed through. I believe that these are the only possible values. Would it be better to accept any, and simply only use these three then?